.github/workflows/workflow.yml

```diff
@@ -139,8 +139,8 @@ jobs:
         run: tox -e package-version-checks
       - name: Check package dependencies
         run: tox -e package-dependencies-checks
-      - name: Check generate protocols
-        run: tox -e check-generate-all-protocols
+      # - name: Check generate protocols
+      #   run: tox -e check-generate-all-protocols
```

Comment:
should this be reintroduced?

Reply:
Yes just on it :)

Reply:
https://github.com/valory-xyz/open-aea/pull/273/commits/75fa58e5621a11af28769f84ad958adfead26412